### PR TITLE
Fixed the Alphabet Transparent header issue.(Bugfix)

### PR DESCRIPTION
**Pull request by i-anuragmishra**
**What kind of change does this PR introduce?**
This PR fixes the issue #476 , That was the Sticky header was transparent and was over lapping the Member list card. It is fixed with this following PR. By giving it a Color. It's a Bugfix.

**Did you add tests for your changes?**
No. It just fixes a UI bug.

**If relevant, did you update the documentation?**
No.

**Summary**
The Header in the  Member list was over lapping due to it being transparent and was causing the issue #476 . This has been resolved By giving it appropriate color and Height.

**Does this PR introduce a breaking change?**
No.

**Other information**
Screenshot_20210327-003217

Here is the screen shot of Improved Member list with out overlapping header.

### CODE REVIEW

**Comment by i-anuragmishra** [comment]
Hey @Sagar2366 This PR solves the Issue #476  , Sorry for the late update, faced some git related issues on my side. Please Review. Thanks.

**Comment by Sagar2366** [comment]
@i-anuragmishra can you please disable auto-formatter and send only changes you added to fix the issue.

**Comment by Sagar2366** [comment]
Screenshot 2021-03-27 at 6 34 51 PM

On scrolling the page, if there is no member to show for the letter, it will still appear on screen.

**Comment by i-anuragmishra** [comment]
> Screenshot 2021-03-27 at 6 34 51 PM
> 
> On scrolling the page, if there is no member to show for the letter, it will still appear on screen.

Yeah, I thought since the Issue only addressed the Transparency  of the Header , so i thought it would not be appropriate for me to push a Commit for that under this PR.

**Comment by i-anuragmishra** [comment]
> Screenshot 2021-03-27 at 6 34 51 PM
> 
> On scrolling the page, if there is no member to show for the letter, it will still appear on screen.

Yeah, I thought since the Issue only addressed the Traspicancy of the Header , so i thought it would not be approcpiarite for me to push a Commit for that under PR.

**Comment by i-anuragmishra** [comment]
> @i-anuragmishra can you please disable auto-formatter and send only changes you added to fix the issue.

okay, but it's just indent changes so it won't affect the code base.

File: lib/views/pages/members/members.dart [hunk 1]
```diff
@@ -173,11 +168,13 @@ class _OrganizationsState extends State<Organizations> {
                 )));
   }
 
-
   //widget which divides the list according to letters
   Widget alphabetDividerList(BuildContext context, List membersList) {
     return SliverStickyHeader(
```
Comment (i-anuragmishra) [inline lib/views/pages/members/members.dart:173]: Here are the actual changes introduced.

**Comment by Sagar2366** [comment]
> > @i-anuragmishra can you please disable auto-formatter and send only changes you added to fix the issue.
> 
> okay, but it's just indent changes so it won't affect the code base.

Though it is just indent changes, but makes reviewing difficult. 
Kindly share the changes which fix the issue, disable auto-formatter.

**Comment by Sagar2366** [comment]
> > Screenshot 2021-03-27 at 6 34 51 PM
> > On scrolling the page, if there is no member to show for the letter, it will still appear on screen.
> 
> Yeah, I thought since the Issue only addressed the Transparency of the Header , so i thought it would not be appropriate for me to push a Commit for that under PR.

Add changes to this PR to fix discussed issue.

**Comment by i-anuragmishra** [comment]
Hey @Sagar2366 done. Indent changes have been reverted. Just but to view just them i need to change the filter and only look at last commit. Thanks .

**Comment by i-anuragmishra** [comment]
Last commit fixed reverted the indent changes. Please  review .

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**